### ISSUE TITLE
why cuda=False

### ISSUE THREAD

**Issue by xysaber** [jfzhang95/pytorch-deeplab-xception#150]
when i use the command "bash train_voc.sh",i found i'm false using cuda and gpu, the namespace shows cuda=False, can anybody solve this issue?

Namespace(backbone='resnet', base_size=513, batch_size=8, checkname='deeplab-resnet', crop_size=513,  **cuda=False**, dataset='pascal', epochs=10, eval_interval=1, freeze_bn=False, ft=False, gpu_ids='0', loss_type='ce', lr=0.007, lr_scheduler='poly', momentum=0.9, nesterov=False, no_cuda=False, no_val=False, out_stride=16, resume=None, seed=1, start_epoch=0, sync_bn=False, test_batch_size=8, use_balanced_weights=False, use_sbd=False, weight_decay=0.0005, workers=4)

**Comment by xysaber** [comment]
i solved it . the reason is that i set the cuda_visible_devices=1, but it should be cude_visible_devices=0, because i have only one GPU for training.
